src/main/java/astor/task/Task.java

```diff
@@ -0,0 +1,104 @@
+package astor.task;
+
+/**
+ * Represents a generic task with a description and status.
+ *
+ * This class provides functionality to mark a task as done or undone, check its status,
+ * and retrieve its description.
+ */
+public class Task {
+    private boolean status;
```

Comment:
Could this be named to sound more like a boolean?
```suggestion
    private boolean isDone;
```